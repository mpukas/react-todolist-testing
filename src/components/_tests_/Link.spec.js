import React from 'react';
import Link from '../Link';
import renderer from 'react-test-renderer';

//snapshot test
it('renders correctly', () => {
  const testFunction = () => true;
  const tree = renderer
    .create(<Link active={true} onClick={testFunction}>Active</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});