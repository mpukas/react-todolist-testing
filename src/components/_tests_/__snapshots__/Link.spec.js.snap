// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  disabled={true}
  onClick={[Function]}
  style={
    Object {
      "marginLeft": "4px",
    }
  }
>
  Active
</button>
`;
